### DISCUSSION TITLE
link:sec/tfjs-backend-cpu ----> what issue is this?

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/human#377] [General]
error code EUNSUPPORTEDPROTOCOL
188 error Unsupported URL Type "link:": link:src/tfjs-backend-cpu
189 verbose exit 1

**Comment by vladmandic** [comment]
you've provided no information on where does this error occur at all.
nor the version in which it occurs.

in general, `human` has dev dependency (dev only) on `@vladmandic/tfjs` which internally uses `link` format which is only supported by some package managers such as `pnpm`, but not all versions of `npm`. but that should not be relevant unless you're trying to rebuild `@vladmandic/tfjs` itself which is not what `human` should do.
